src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8694,6 +8696,52 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/rule-sets/{ruleSetId}':
+    put:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Update a rule set
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer versions.
+
+        Update a rule set including setting the order of the rules within it.
+      operationId: updateRuleSet
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeFolderIdParam'
+        - $ref: '#/parameters/ruleSetIdParam'
+        - $ref: '#/parameters/fieldsParam'
+        - $ref: '#/parameters/ruleSetMinimalEntryIncludeParam'
+        - in: body
+          name: ruleSetBodyUpdate
+          description: The JSON representing the rule set to be updated.
+          required: true
+          schema:
+            $ref: '#/definitions/RuleSetBodyUpdate'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RuleSetEntry'
+        '400':
+          description: |
+            Invalid parameter: **nodeId** or **ruleSetId** is not a valid format, or the node is not the correct type
```

Comment:
What happens if a `ruleIds` array is provided but isn't complete (i.e. some rule ids that exist on this rule set are not present in the newly provided array, or some extra ids that don't exist are provided). Should we include this case in the description of a 400 error?